microsoft/botframework-solutions
# [Botskills] Test structure refactor

## Description

- Update botskills test structure to follow the structure of the rest of the solutions

## Testing Steps

1. Go to `botframework-solutions\lib\typescript\botskills`
1. Check the new structure for the tests
1. Run `npm install` and `npm run build`
1. Run `npm run test` and check that all the tests are passing successfully

## Checklist
